pkg/cmd/grafana-cli/main.go

```diff
@@ -18,13 +17,13 @@ var version = "master"
 func main() {
 	setupLogging()
 
-	flag.Parse()
 	app := cli.NewApp()
 	app.Name = "Grafana cli"
 	app.Usage = ""
 	app.Author = "Grafana Project"
 	app.Email = "https://github.com/grafana/grafana"
 	app.Version = version
+
```

Comment:
Nit-pick: is this change necessary?

Reply:
Sadly 🙈 , this is a cherrypick of an already introduced change. I don't feel comfortable changing anything from this PR without changing it on the master branch.

That being said, I feel like it reads a bit better, but I can see the argument going both ways. 😄